docs/src/pages/components/icon.mdx

```diff
@@ -21,7 +21,7 @@ There are currently <IconCount /> icons available. Note that you can try any of
 
 #### Live Code Editor
 
-```tsx
+```tsx live
```

Comment:
I may have missed this in a different PR, but do we document what `live` does somewhere?

Reply:
No, but I have been thinking about putting together a style guide for documentation and including an explanation on that. For now it's word of mouth. I can add a comment and example use to the code editor component for the time being.

Reply:
I'm content adding a comment in the `Page` component (I think that's where that lives) calling this out and calling that documented.

Reply:
Sounds good! Thanks for clarifying 👍

Reply:
Added a fairly detailed example at the top of the `CodePreview` component showing use cases.